### DISCUSSION TITLE
OrtRun() error

### DISCUSSION THREAD

**Discussion by shrestha-pranita** [microsoft/onnxruntime#13005] [Other Q&A]
After passing feed to session.run(feeds) I keep getting the following error:

2022-09-19 14:28:17.601300 [E:onnxruntime:, sequential_executor.cc:368 Execute] Non-zero status code returned while running Conv node. Name:'Conv_81' Status Message: Invalid input shape: {1}

What does this error mean? Can anyone help me with it?

Also I am trying to use the onnxruntime-web for js implementation

**Comment by pranavsharma** [comment]
Do you see this error only when using onnxruntime-web? Have you tried ORT's Python API?

**Reply by shrestha-pranita** [reply]
The python one works fine. I am getting this error when I try to use ORT in js.

**Reply by pranavsharma** [reply]
In that case please file an issue mentioning this.

**Reply by shrestha-pranita** [reply]
Okay. Can you tell me what Name:'Conv_81' Status Message: Invalid input shape: {1} this error exactly means? I was wondering if the input that I am passing as my input for the feed is not in the correct shape?
